diff -urN analog-4.16.org/analog.cfg analog-4.16/analog.cfg
--- analog-4.16.org/analog.cfg	Tue Feb 13 16:09:08 2001
+++ analog-4.16/analog.cfg	Mon Feb 26 21:04:14 2001
@@ -9,9 +9,10 @@
 # There is a much more extensive configuration file in examples/big.cfg
 #
 # If you need a LOGFORMAT command it must go here, above the LOGFILE command.
-# LOGFILE logfile.log
+LOGFILE /var/log/httpd/access_log
+LANGUAGE POLISH
 # OUTFILE outfile.html
-# HOSTNAME "[my organisation]"
+HOSTNAME "[PLD Linux Server]"
 # REQINCLUDE pages
 LINKINCLUDE pages
 REFLINKINCLUDE *
diff -urN analog-4.16.org/anlgform.html analog-4.16/anlgform.html
--- analog-4.16.org/anlgform.html	Tue Feb 13 16:09:08 2001
+++ analog-4.16/anlgform.html	Mon Feb 26 21:06:02 2001
@@ -3,8 +3,7 @@
 </head>
 
 <body>
-<h1><!-- img src="/images/analogo.gif" alt="" --> Analog form interface</h1>
-
+<h1><img src="/icons/analogo.gif" alt="Analog" --> Analog form interface</h1>
 <!-- Make the edits below. You also need to edit anlgform.pl. -->
 
 <!-- First, uncomment the img tag above, edited if necessary. -->
@@ -24,11 +23,11 @@
 
 <!-- Many systems will want the IMAGEDIR to be different on the form -->
 <!-- than from the command line, because it should not be within /cgi-bin/ -->
-<!-- input type=hidden name="IMAGEDIR" value="/images/" -->
+<!-- input type=hidden name="IMAGEDIR" value="/icons/" -->
 
 <!-- Some users will want to set their logfile like this. If you do, you -->
 <!-- musn't also specify the LOGFILE in the "cg" configuration file above. -->
-<!-- input type=hidden name="LOGFILE" value="/usr/local/etc/httpd/logs/access_log" -->
+<!-- input type=hidden name="LOGFILE" value="/usr/var/httpd/log/access_log" -->
 
 <!-- After you've made these edits, you can remove the "strong" lines below -->
 
diff -urN analog-4.16.org/anlgform.pl analog-4.16/anlgform.pl
--- analog-4.16.org/anlgform.pl	Tue Feb 13 16:09:08 2001
+++ analog-4.16/anlgform.pl	Mon Feb 26 21:03:22 2001
@@ -19,6 +19,7 @@
 #    of Perl (don't remove the #! though).
 # 3) You also need to edit anlgform.html if you want to use the form.
 # 4) Add to the forbidden commands below if you want.
+$analog = '/usr/bin/analog';
 
 @forbidden = qw(LOGFORMAT APACHELOGFORMAT DEFAULTLOGFORMAT
 		APACHEDEFAULTLOGFORMAT HEADERFILE FOOTERFILE UNCOMPRESS
diff -urN analog-4.16.org/anlghea2.h analog-4.16/anlghea2.h
--- analog-4.16.org/anlghea2.h	Tue Feb 13 16:09:08 2001
+++ analog-4.16/anlghea2.h	Mon Feb 26 20:59:43 2001
@@ -50,16 +50,16 @@
 #endif
 
 #ifndef LANGDIR
-#define LANGDIR ANALOGDIR"lang/"
+#define LANGDIR ANALOGDIR "/usr/share/analog/lang/"
 /* Directory where the language files live. */
 #endif
 
 #ifndef NODNS
-#define DNSFILE ANALOGDIR"dnscache"
+#define DNSFILE ANALOGDIR "/var/lib/analog/dnscache"
 /* where to store resolved addresses so we don't have to look them up next
    time */
 
-#define DNSLOCKFILE ANALOGDIR"dnslock"
+#define DNSLOCKFILE ANALOGDIR "/var/lock/dnslock"
 /* A lock file to stop two processes trying to write the DNSFILE at once */ 
 
 #define DNS (DNS_NONE)
diff -urN analog-4.16.org/anlghead.h analog-4.16/anlghead.h
--- analog-4.16.org/anlghead.h	Tue Feb 13 16:09:09 2001
+++ analog-4.16/anlghead.h	Mon Feb 26 21:02:30 2001
@@ -19,19 +19,19 @@
 #define ANLGHEAD
 
 #ifndef HOSTNAME
-#define HOSTNAME "[my organisation]"
+#define HOSTNAME "[PLD Linux Server]"
 /* the name of your organisation or WWW host. This is used for printing
    at the top of the output */
 #endif
 
 #ifndef HOSTURL
-#define HOSTURL "none"
+#define HOSTURL "/"
 /* the URL of your host's home page, for linking to at the top of the
    output; use "none" for no linking. */
 #endif
 
 #ifndef ANALOGDIR
-#define ANALOGDIR "/usr/local/analog/analog-4.16/"
+#define ANALOGDIR ""
 #endif
 /* The directory where your analog stuff lives (or will live). You need to
    get this right otherwise analog won't be able to find any of its auxiliary
@@ -41,18 +41,18 @@
    which means the file analog.cfg within the ANALOGDIR. */
 
 #ifndef LOGFILE
-#define LOGFILE "/usr/local/apache/logs/access_log"
+#define LOGFILE "/var/log/httpd/access_log"
 /* The name of the default logfile. Can be a list, separated by commas
    (but no spaces), and can include wild cards. */
 #endif
 
 #ifndef OUTFILE
-#define OUTFILE "stdout"
+#define OUTFILE "/home/httpd/html/usage/analog.html"
 /* The default file for output. Use "stdout" for stdout. */
 #endif
 
 #ifndef IMAGEDIR
-#define IMAGEDIR "images/"
+#define IMAGEDIR "/icons/"
 /* URL of the directory where the images for the graphical reports live.
    The URL can be absolute, or relative to the output page: e.g., just the
    empty string "" for the same directory as the output page. */
@@ -71,7 +71,7 @@
 #endif
 
 #ifndef DEFAULTCONFIGFILE
-#define DEFAULTCONFIGFILE ANALOGDIR"analog.cfg"
+#define DEFAULTCONFIGFILE ANALOGDIR "/etc/analog.cfg"
 /* the name of the default configuration file; see README.html for a
    description of this file. Use "none" (NOT ANALOGDIR"none"!) for no
    configuration file. */
