diff -urN analog4.02.orig/analog.cfg analog4.02/analog.cfg
--- analog4.02.orig/analog.cfg	Sat Jan 29 17:00:59 2000
+++ analog4.02/analog.cfg	Fri Feb  4 00:12:37 2000
@@ -7,7 +7,8 @@
 # There is a much more extensive configuration file in examples/big.cfg
 #
 # If you need a LOGFORMAT command it must go here, above the LOGFILE command.
-# LOGFILE logfile.log
+LOGFILE /var/log/httpd/access_log
+LANGUAGE POLISH
 # HOSTNAME "[my organisation]"
 # REQINCLUDE pages
 LINKINCLUDE pages
diff -urN analog4.02.orig/anlgform.html analog4.02/anlgform.html
--- analog4.02.orig/anlgform.html	Mon Nov  8 20:33:04 1999
+++ analog4.02/anlgform.html	Fri Feb  4 00:12:38 2000
@@ -3,8 +3,7 @@
 </head>
 
 <body>
-<h1><!-- img src="/images/analogo.gif" alt="" --> Analog form interface</h1>
-
+<h1><img src="/icons/analogo.gif" alt="Analog" --> Analog form interface</h1>
 <!-- Make the edits below. You also need to edit anlgform.pl. -->
 
 <!-- First, uncomment the img tag above, edited if necessary. -->
@@ -24,10 +23,10 @@
 
 <!-- Many systems will want the IMAGEDIR to be different on the form... -->
 <!-- than from the commandline, because it should not be within /cgi-bin/ -->
-<!-- input type=hidden name="IMAGEDIR" value="/images/" -->
+<!-- input type=hidden name="IMAGEDIR" value="/icons/" -->
 
 <!-- Some users will want to set their logfile like this -->
-<!-- input type=hidden name="LOGFILE" value="/usr/local/etc/httpd/logs/access_log" -->
+<!-- input type=hidden name="LOGFILE" value="/usr/var/httpd/log/access_log" -->
 
 <hr>
 <h2>1. Report choices</h2>
diff -urN analog4.02.orig/anlghea2.h analog4.02/anlghea2.h
--- analog4.02.orig/anlghea2.h	Sat Jan 29 17:00:59 2000
+++ analog4.02/anlghea2.h	Fri Feb  4 00:12:38 2000
@@ -48,16 +48,16 @@
 #endif
 
 #ifndef LANGDIR
-#define LANGDIR ANALOGDIR"lang/"
+#define LANGDIR ANALOGDIR "/usr/share/analog/lang/"
 /* Directory where the language files live. */
 #endif
 
 #ifndef NODNS
-#define DNSFILE ANALOGDIR"dnscache"
+#define DNSFILE ANALOGDIR "/var/state/analog/dnscache"
 /* where to store resolved addresses so we don't have to look them up next
    time */
 
-#define DNSLOCKFILE ANALOGDIR"dnslock"
+#define DNSLOCKFILE ANALOGDIR "/var/lock/dnslock"
 /* A lock file to stop two processes trying to write the DNSFILE at once */ 
 
 #define DNS (DNS_NONE)
diff -urN analog4.02.orig/anlghead.h analog4.02/anlghead.h
--- analog4.02.orig/anlghead.h	Sat Jan 29 17:01:09 2000
+++ analog4.02/anlghead.h	Fri Feb  4 00:12:39 2000
@@ -19,19 +19,19 @@
 #define ANLGHEAD
 
 #ifndef HOSTNAME
-#define HOSTNAME "[my organisation]"
+#define HOSTNAME "[PLD Linux Server]"
 /* the name of your organisation or WWW host. This is used for printing
    at the top of the output */
 #endif
 
 #ifndef HOSTURL
-#define HOSTURL "none"
+#define HOSTURL "/"
 /* the URL of your host's home page, for linking to at the top of the
    output; use "none" for no linking. */
 #endif
 
 #ifndef ANALOGDIR
-#define ANALOGDIR "/usr/local/analog4.02/"
+#define ANALOGDIR ""
 #endif
 /* The directory where your analog stuff lives (or will live). You need to
    get this right otherwise analog won't be able to find any of its auxiliary
@@ -41,18 +41,18 @@
    which means the file analog.cfg within the ANALOGDIR. */
 
 #ifndef LOGFILE
-#define LOGFILE "/usr/local/apache/logs/access_log"
+#define LOGFILE "/var/log/httpd/access_log"
 /* The name of the default logfile. Can be a list, separated by commas
    (but no spaces), and can include wild cards. */
 #endif
 
 #ifndef OUTFILE
-#define OUTFILE "stdout"
+#define OUTFILE "/home/httpd/html/usage/analog.html"
 /* The default file for output. Use "stdout" for stdout. */
 #endif
 
 #ifndef IMAGEDIR
-#define IMAGEDIR "images/"
+#define IMAGEDIR "/icons/"
 /* URL of the directory where the images for the graphical reports live.
    The URL can be absolute, or relative to the output page: e.g., just the
    empty string "" for the same directory as the output page. */
@@ -71,7 +71,7 @@
 #endif
 
 #ifndef DEFAULTCONFIGFILE
-#define DEFAULTCONFIGFILE ANALOGDIR"analog.cfg"
+#define DEFAULTCONFIGFILE ANALOGDIR "/etc/analog.cfg"
 /* the name of the default configuration file; see README.html for a
    description of this file. Use "none" (NOT ANALOGDIR"none"!) for no
    configuration file. */
