diff -urN analog-5.22/anlgform.html analog-5.22.new/anlgform.html
--- analog-5.22/anlgform.html	Wed Mar 20 10:38:41 2002
+++ analog-5.22.new/anlgform.html	Fri Apr  5 12:38:44 2002
@@ -3,7 +3,7 @@
 </head>
 
 <body>
-<h1><!-- img src="/images/analogo.gif" alt="" --> Analog form interface</h1>
+<h1><img src="/images/analogo.gif" alt="" --> Analog form interface</h1>
 
 <!-- Make the edits below. You also need to edit anlgform.pl. -->
 
@@ -15,10 +15,12 @@
 
 <!-- Many systems will want the IMAGEDIR to be different on the form -->
 <!-- than from the command line, because it should not be within /cgi-bin/ -->
-<!-- input type=hidden name="IMAGEDIR" value="/images/" -->
+<!-- input type=hidden name="IMAGEDIR" value="/icons/" -->
 
 <!-- Some users will want to set their logfile like this. -->
 <!-- input type=hidden name="LOGFILE" value="/usr/local/etc/httpd/logs/access_log" -->
+<!-- input type=hidden name="LOGFILE" value="/usr/var/httpd/log/access_log" -->
+
 
 <!-- After you've made these edits, you can remove the "strong" lines below -->
 
diff -urN analog-5.22/anlgform.pl analog-5.22.new/anlgform.pl
--- analog-5.22/anlgform.pl	Wed Mar 20 10:38:41 2002
+++ analog-5.22.new/anlgform.pl	Fri Apr  5 12:39:39 2002
@@ -19,6 +19,7 @@
 #    of Perl (don't remove the #! though).
 # 3) You also need to edit anlgform.html if you want to use the form.
 # 4) Add to the forbidden commands below if you want.
+$analog='/usr/bin/analog'
 
 @forbidden = qw(LOGFORMAT APACHELOGFORMAT DEFAULTLOGFORMAT
 		APACHEDEFAULTLOGFORMAT HEADERFILE FOOTERFILE UNCOMPRESS
Binary files analog-5.22/src/alias.o and analog-5.22.new/src/alias.o differ
Binary files analog-5.22/src/analog.o and analog-5.22.new/src/analog.o differ
diff -urN analog-5.22/src/anlghea2.h analog-5.22.new/src/anlghea2.h
--- analog-5.22/src/anlghea2.h	Wed Mar 20 10:38:51 2002
+++ analog-5.22.new/src/anlghea2.h	Fri Apr  5 12:40:33 2002
@@ -50,11 +50,12 @@
 #endif
 
 #ifndef NODNS
-#define DNSFILE "dnscache"
+#define DNSFILE "/var/lib/analog/dnscache"
 /* where to store resolved addresses so we don't have to look them up next
    time */
 
-#define DNSLOCKFILE "dnslock"
+#define DNSLOCKFILE "/var/lock/dnslock"
+
 /* A lock file to stop two processes trying to write the DNSFILE at once */ 
 
 #define DNS (DNS_NONE)
diff -urN analog-5.22/src/anlghead.h analog-5.22.new/src/anlghead.h
--- analog-5.22/src/anlghead.h	Wed Mar 20 10:38:52 2002
+++ analog-5.22.new/src/anlghead.h	Fri Apr  5 12:42:55 2002
@@ -19,30 +19,31 @@
 #define ANLGHEAD_H
 
 #ifndef HOSTNAME
-#define HOSTNAME "[my organisation]"
+#define HOSTNAME "[PLD Linux Server]"
 /* The name of your organisation or web server. This is used for printing
    at the top of the output. */
 #endif
 
 #ifndef HOSTURL
-#define HOSTURL "none"
+#define HOSTURL "/"
 /* The URL of your host's home page, for linking to at the top of the
    output; use "none" for no linking. */
 #endif
 
 #ifndef LOGFILE
-#define LOGFILE "logfile.log"
+#define LOGFILE "/var/log/httpd/access_log"
 /* The name of the default logfile. Can be a list, separated by commas
    (but no spaces), and can include wild cards. */
 #endif
 
 #ifndef OUTFILE
-#define OUTFILE "stdout"
+#define OUTFILE "/home/httpd/html/usage/analog.html"
+
 /* The default file for output. Use "stdout" for stdout. */
 #endif
 
 #ifndef IMAGEDIR
-#define IMAGEDIR "images/"
+#define IMAGEDIR "/icons/"
 /* URL of the directory where the images for the graphical reports live.
    The URL can be absolute, or relative to the output page: e.g., just the
    empty string "" for the same directory as the output page. */
@@ -61,7 +62,7 @@
 #endif
 
 #ifndef DEFAULTCONFIGFILE
-#define DEFAULTCONFIGFILE "analog.cfg"
+#define DEFAULTCONFIGFILE "/etc/analog.cfg"
 /* The name of the default configuration file. */
 #endif
 
@@ -82,7 +83,7 @@
    directory. This may or may not work, so it's better to specify a location
    explicitly here if you know where the files will be kept. */
 #ifndef LANGDIR
-#define LANGDIR NULL
+#define LANGDIR "/usr/share/analog/lang/"
 #endif
 /* Directory where the language files live. Actually, if this one is defined
    to be NULL, they will be looked for inside the "lang" subdirectory of the
